From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

The '-T' flag needs to be passed to the RTS in order for tests to pass
when the 'bounded_memory' flag is enabled.

--- hnix-store-core-0.5.0.0/hnix-store-core.cabal
+++ hnix-store-core-0.5.0.0-new/hnix-store-core.cabal
@@ -107,7 +102,7 @@
   import: commons
   if flag(bounded_memory)
     cpp-options: -DBOUNDED_MEMORY
-    ghc-options: -rtsopts -fprof-auto
+    ghc-options: -rtsopts -fprof-auto -with-rtsopts=-T
   type: exitcode-stdio-1.0
   main-is: Driver.hs
   other-modules:
